### PR TITLE
coaches should see projects

### LINKED ISSUE #402: coaches & student participations
There are 2 options:
- make projects/id accessible by coaches, so that coaches can view the name of the project the student was assigned to
- show in frontend "student has a participation", but don't show the project they've been assigned to

right now there is simply no way to see that a student has a participation if the coach doesn't have access to the project the student was assigned to.

### PR DESCRIPTION
**Pull request by QuintenVervynck**
Fixes #402

### CODE REVIEW

File: backend/app/routers/editions.py [hunk 1]
```diff
@@ -198,20 +198,15 @@ async def get_edition_students(year: int, orderby: str = "", search: str = "", s
 
 
 @router.get("/{year}/projects", response_description="Projects retrieved")
-async def get_edition_projects(year: int, role: RoleChecker(UserRole.COACH) = Depends(), session: AsyncSession = Depends(get_session), Authorize: AuthJWT = Depends()):
+async def get_edition_projects(year: int, role: RoleChecker(UserRole.COACH) = Depends(), session: AsyncSession = Depends(get_session)):
```
Comment (StefVdHaute) [inline backend/app/routers/editions.py:201]: role is unused argument.

**Review by StefVdHaute — CHANGES_REQUESTED** [review]
Only unused argument, all the rest is good.

**Comment by QuintenVervynck** [comment]
I moved it to the dependency list, that way it isn't listed as an argument (we still needed to check the coach, so we can't just remove it)

**Comment by QuintenVervynck** [comment]
seems like we fixed it twice hahaha

**Comment by StefVdHaute** [comment]
2 different endpoints haha

**Review by StefVdHaute — APPROVED** [review]

**Pull request merged**